cmd/ingester/app/flags.go

```diff
@@ -103,6 +137,39 @@ func AddFlags(flagSet *flag.FlagSet) {
 		ConfigPrefix+SuffixDeadlockInterval,
 		DefaultDeadlockInterval,
 		"Interval to check for deadlocks. If no messages gets processed in given time, ingester app will exit. Value of 0 disables deadlock check.")
+	flagSet.String(
+		KafkaConsumerConfigPrefix+SuffixAuthentication,
+		DefaultAuthentication,
+		"Authentication type used to authenticate with kafka cluster")
```

Comment:
Could you include the valid values (e.g. none, kerberos) and which is default.

Reply:
I changed this to indicate valid values, the default value is displayed by the library that generates the help output.

This is the printed message:

`      --kafka.consumer.authentication string            Authentication type used to authenticate with kafka cluster. e.g. none, kerberos (default "none")
`